core/src/main/resources/reference.conf

```diff
@@ -1385,7 +1385,7 @@ datastax-java-driver {
       # Required: yes
       # Modifiable at runtime: no
       # Overridable in a profile: no
-      tick-duration = 100 milliseconds
+      tick-duration = 1 milliseconds
```

Comment:
Small nit: Config also support the singular form, so `1 millisecond` is also valid and looks prettier 😃

Reply:
I completely overlooked that, good catch.